rdmo/conditions/tests/test_viewset_condition.py

```diff
@@ -18,16 +18,19 @@
         'editor': 200, 'reviewer': 200, 'api': 200, 'user': 403, 'anonymous': 401
     },
     'detail': {
-        'editor': 200, 'reviewer': 200, 'api': 200, 'user': 403, 'anonymous': 401
+        'editor': 200, 'reviewer': 200, 'api': 200, 'user': 404, 'anonymous': 401
     },
     'create': {
         'editor': 201, 'reviewer': 403, 'api': 201, 'user': 403, 'anonymous': 401
     },
     'update': {
-        'editor': 200, 'reviewer': 403, 'api': 200, 'user': 403, 'anonymous': 401
+        'editor': 200, 'reviewer': 403, 'api': 200, 'user': 404, 'anonymous': 401
     },
     'delete': {
-        'editor': 204, 'reviewer': 403, 'api': 204, 'user': 403, 'anonymous': 401
+        'editor': 204, 'reviewer': 403, 'api': 204, 'user': 404, 'anonymous': 401
+    },
+    'copy': {
```

Comment:
We need this copy entry or we delete the copy tests all together since we don't need them in the new data model anymore.

Reply:
think it would be better to add this `copy` entry and keep the tests for copy via the api.. maybe there are users that (will) use this feature?